app/controllers/topical_events_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
   def show
     @classification = TopicalEvent.friendly.find(params[:id])
-    @policies = @classification.published_policies.includes(:translations, :document)
+    @policies = @classification.published_policies
```

Comment:
If we're not showing policies on the site anymore, do we need to be loading them here?